resources/js/bootstrap.js

```diff
@@ -27,6 +27,8 @@ import Vue from 'vue'
 import VueCookies from 'vue-cookies'
 import PmqlInput from "./components/shared/PmqlInput.vue";
 import GlobalSearch from "./components/shared/GlobalSearch.vue";
+import DataTreeToggle from "./components/common/data-tree-toggle.vue";
+import TreeView from "./components/TreeView.vue";
```

Comment:
@gustavobascope If we are going to include this globally can we remove it from resources/js/requests/show.js and resources/js/tasks/show.js ?